src/index.html

```diff
@@ -123,7 +129,8 @@ <h1 class="has-text-weight-bold">SankeyDiagram.net</h1>
                         <div class="divider">Share</div>
                         <span class="button is-dark p-1 mb-1 has-shadow download-as-png-button is-clickable level-item"><span class="material-icons mr-1">get_app</span> Download as PNG</span>
                         <span class="button is-dark p-1 mb-1 has-shadow download-as-svg-button is-clickable level-item"><span class="material-icons mr-1">get_app</span> Download as SVG</span>
-                        <span class="button is-dark p-1 has-shadow copy-link-button is-clickable level-item" data-tooltip="Copied!"><span class="material-icons mr-1">content_copy</span> Get Share Link</span>
+                        <span class="button is-dark p-1 mb-1 has-shadow copy-link-button is-clickable level-item" data-tooltip="Copied!"><span class="material-icons mr-1">content_copy</span> Get Share Link</span>
+                        <span class="button is-dark p-1 has-shadow export-txt-button is-clickable level-item"><span class="material-icons mr-1">file_download</span> Export as TXT file</span>
```

Comment:
Here I would also propose replacing  "Export as TXT file" with "Export as Text File"

Reply:
Sure, no problem. I wanted to make it consistent with the PNG and SVG button texts.